src/auth/utils.ts

```diff
@@ -53,8 +53,12 @@ import { isValidCodeWhispererCoreConnection } from '../codewhisperer/util/authUt
 // TODO: Look to do some refactoring to handle circular dependency later and move this to ./commands.ts
 export const showConnectionsPageCommand = 'aws.auth.manageConnections'
 
-export async function promptForConnection(auth: Auth, type?: 'iam' | 'sso'): Promise<Connection | void> {
-    const resp = await createConnectionPrompter(auth, type).prompt()
+export async function promptForConnection(
+    auth: Auth,
+    type?: 'iam' | 'sso',
+    excludeSso?: boolean
```

Comment:
why can't `type = 'foo'` be used instead of adding a new `excludeSso` parameter ?

Reply:
I can add an "iam-only" type, just didn't want to change existing behavior outside of Composer.

Reply:
something like "iam-only" would be preferred. cc @nkomonen-amazon